packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```diff
@@ -0,0 +1,394 @@
+              "Principal": {
+                "AWS": {
+                  "Ref": "AWS::AccountId"
+                }
+              }
+            },
+            {
+              "Fn::If": [
+                "HasTrustedPrincipals",
+                {
+                  "Action": "sts:AssumeRole",
+                  "Effect": "Allow",
+                  "Principal": {
+                    "AWS": {
+                      "Ref": "TrustedPrincipals"
+                    }
+                  }
+                },
+                {
+                  "Ref": "AWS::NoValue"
+                }
+              ]
+            }
+          ]
+        },
+        "Policies": [
+          {
+            "PolicyDocument": {
+              "Statement": [
+                {
+                  "Action": [
+                    "cloudformation:CreateChangeSet",   "cloudformation:DeleteChangeSet",
+                    "cloudformation:DescribeChangeSet", "cloudformation:DescribeStacks",
+                    "cloudformation:ExecuteChangeSet",
+                    "s3:GetObject*",                    "s3:GetBucket*",
+                    "s3:List*",                         "s3:Abort*",
+                    "s3:DeleteObject*",                 "s3:PutObject*",
+                    "kms:Decrypt",                      "kms:DescribeKey"
+                  ],
+                  "Resource": "*",
+                  "Effect": "Allow"
+                },
+                {
+                  "Action": "iam:PassRole",
+                  "Resource": {
+                    "Fn::Sub": "${CloudFormationExecutionRole.Arn}"
+                  },
+                  "Effect": "Allow"
+                }
+              ],
+              "Version": "2012-10-17"
+            },
+            "PolicyName": "default"
+          }
+        ],
+        "RoleName": {
+          "Fn::Sub": "cdk-bootstrap-hnb659fds-deployment-action-role-${AWS::AccountId}-${AWS::Region}"
+        }
+      },
+      "Condition": "HasTrustedPrincipals"
+    },
+    "CloudFormationExecutionRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Action": "sts:AssumeRole",
+              "Effect": "Allow",
+              "Principal": {
+                "Service": "cloudformation.amazonaws.com"
+              }
+            }
+          ]
+        },
+        "ManagedPolicyArns": {
+          "Ref": "CloudFormationExecutionPolicies"
+        },
+        "RoleName": {
+          "Fn::Sub": "cdk-bootstrap-hnb659fds-cloudformation-execution-role-${AWS::AccountId}-${AWS::Region}"
+        }
+      },
+      "Condition": "HasTrustedPrincipals"
+    }
+  },
+  "Outputs": {
+    "BucketName": {
+      "Description": "The name of the S3 bucket owned by the CDK toolkit stack",
+      "Value": { "Fn::Sub":  "${StagingBucket}" },
+      "Export": {
+        "Name": { "Fn::Sub": "${AWS::StackName}:BucketName" }
```

Comment:
Do we strictly need these exports?

Reply:
I think we should omit them.

Reply:
The exports are required. See here: https://github.com/aws/aws-cdk/blob/master/design/cdk-bootstrap.md#cli-old-framework-old-bootstrap-new-init-template-old for details.

Reply:
But that mentions backwards compatibility with old CLIs, and the old CLI reads the Outputs, not necessarily the Exports. My question was around the exports.

Reply:
Why do we have these Exports in the old bootstrapping?

Reply:
Actually, it turns out we *don't* have those in the old bootstrapping 🤐.

That's weird - I could swear I copy-pasted this entire section from the existing bootstrap template, but I guess I didn't.

Will remove the exports.